src/libraries/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/System.Diagnostics.FileVersionInfo.Tests.csproj

```diff
@@ -50,5 +50,8 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </ProjectReference>
     <ProjectReference Include="..\System.Diagnostics.FileVersionInfo.TestAssembly\System.Diagnostics.FileVersionInfo.TestAssembly.csproj" Condition="'$(TargetOS)' == 'browser'" />
+    <!-- R2R testing does not tolerate the combination of a regular project reference and a content reference. -->
+    <!-- This is actually a bug in the SDK and ought to be fixed there. -->
```

Comment:
Could you add a link to the tracking SDK bug?

Reply:
I believe we don't have one. Will have to file it.

Reply:
Did you file the bug? If so could you add a link here same as we have on line 46 (for a similar issue)?